proto/stakeibc/deposit_record.proto

```diff
@@ -6,13 +6,15 @@ option go_package = "github.com/Stride-Labs/stride/x/stakeibc/types";
 message DepositRecord {
   uint64 id = 1;
   int64 amount = 2; 
-  string denom = 3; 
+  string denom = 3;
   string hostZoneId = 4; 
-  string sender = 5; 
-  enum Purpose {
-    RECEIPT = 0;
-    TRANSACTION = 1;
+  enum Status {
+    // pending transfer to delegate account
+    TRANSFER = 0;
+    // pending staking on delegate account
+    STAKE = 1;
   }
-  Purpose purpose = 6; 
-  
+  Status status = 6; 
+
+  reserved 5;
```

Comment:
I haven't been using reserved when I fiddle with other protos, do I need to go back and do this even if I delete *.pb.go's?

Reply:
The short answer is that this will always work, while not reserving / updating fields live will sometimes break. Easier just to reserve fields!

If we start with a clean chain state, we don't need this. Once we're updating a live chain, we usually will.